plugins/database/postgresql/postgresql.go

```diff
@@ -427,10 +451,10 @@ func (p *PostgreSQL) defaultDeleteUser(ctx context.Context, username string) err
 
 	// can't drop if not all privileges are revoked
 	if rows.Err() != nil {
-		return errwrap.Wrapf("could not generate revocation statements for all rows: {{err}}", rows.Err())
+		return fmt.Errorf("could not generate revocation statements for all rows: %w", rows.Err())
```

Comment:
Looks like we're [using](https://github.com/hashicorp/vault/blob/master/go.mod#L56) `errwrap` v1.0.0 which doesn't support upwrapping errors wrapped by `%w`. We should update go.mod first (probably as a separate PR) before merging this in.

Reply:
Updated